infrastructure/docker/celo/start.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+
+mkdir -p $CELO_DATA_DIR/keystore
+cp $CELO_INIT_DIR/keystore/* $CELO_DATA_DIR/keystore
+cp $CELO_INIT_DIR/password.txt $CELO_DATA_DIR/password.txt
+[ ! -d "$CELO_DATA_DIR/celo" ] && geth --nousb --datadir=$CELO_DATA_DIR init $CELO_INIT_DIR/genesis.json
```

Comment:
Would it be worth printing a message if we get into chain initialization?

Reply:
Celo/Geth is printing information in logs:

> INFO [02-11|19:32:23.105] Initialised chain configuration          config="{ChainID: 1101 Homestead: 0 DAO: <nil> DAOSupport: false EIP150: 0 EIP155: 0 EIP158: 0 Byzantium: 0 Constantinople: 0 Petersburg: 0 Istanbul: 0 Churrito: <nil>, Donut: <nil>, Engine: istanbul}"

Do you think it is not enough?

Reply:
Yep I noticed this line finally. It gets lost very easy in this output. But after reconsideration I say leave it as is.